packages/vda-node-client/src/index.ts

```diff
@@ -0,0 +1,6 @@
+export * from './getBalance'
+export * from './getContractDecimal'
+export * from './getDataCenters'
+export * from './getLogInfos'
+export * from './getNodes'
+export * from './getStakeInfo'
```

Comment:
We need a `VdaNodeClient()` object that takes an optional `rpcUrl` parameter and exposes all these methods.

Reply:
We can create `vda-node-manager` instance in `read-only` mode.
Pls check the line#18~line#21 in the `vda-node-manager/test/user_read.test.ts`

Reply:
This doesn't help.

1/ We need the ability to customize the RPC_URL
2/ I'm trying to keep the code structure the same as the other libraries, where we create an object to interact with the smart contracta

Reply:
Created a class.